tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -21,6 +22,16 @@ interface AssertedSchema extends MigrateConvergedPkgGeneratorSchema {
   name: string;
 }
 
+jest.mock(
```

Comment:
mock away whole `enquirer` for sake of tests. 
- only `prompt` is being used within our implementation
- scoped mock overrides are collocated within test suites